### PR TITLE
Skip creating secret in context preparer

### PR DESCRIPTION
**Pull request by r2d4**
We do this in the pod_spec_mutator functions now

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/159)

### CODE REVIEW

File: fairing/builders/cluster/gcs_context.py [hunk 1]
```diff
@@ -30,29 +29,9 @@ def upload_context(self, context_filename):
                     bucket_name=self.gcp_project,
                     blob_name='fairing_builds/' + context_hash,
                     file_to_upload=context_filename)
-        
-    def create_secret(self):
```
Comment (abhi-g) [inline fairing/builders/cluster/gcs_context.py]: @r2d4 Is it the case that based on our analysis of all paths a user may take (local/KF/hosted notebooks), there won't be a need to create secret?

Comment (r2d4) [reply fairing/builders/cluster/gcs_context.py]: The (possible) secret generation was moved into a pod_spec_mutator - the GCS context preparer shouldn't be responsible for creating this secret.

**Comment by abhi-g** [comment]
you might need to self-approve

**Pull request merged**